sdk/identity/identity/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- The `ManagedIdentityCredential` now suppports specifying the Client ID through the `AZURE_MANAGED_IDENTITY_CLIENT_ID` environment variable.
```

Comment:
Should we use `AZURE_CLIENT_ID` instead (DAC already does this in its options)

Reply:
Using AZURE_CLIENT_ID would be a breaking change. Let me send you an email that @g2vinay sent last week.

Reply:
Oh - yes I remember it now.. Thanks!